clang/unittests/AST/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ add_clang_unittest(ASTTests
   CommentLexer.cpp
   CommentParser.cpp
   CommentTextTest.cpp
+  ConceptPrinterTest.cpp
```

Comment:
Note: we probably want to update the gn build as well: https://github.com/llvm/llvm-project/tree/main/llvm/utils/gn

Reply:
Done (we're not required to update gn build files, and there's a bot that can automate the porting).